### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by Sherrylone** [sthalles/SimCLR#13]
Hi.
Thanks for your great work. But I have a little confusion. You implement the contrastive loss by Cross-Entropy Loss without softmax function. So the negatives actually didn't work but only positives.

**Comment by nzw0301** [comment]
@Sherrylone The softmax function is applied in [`torch.nn.CrossEntropyLoss`](https://pytorch.org/docs/master/generated/torch.nn.CrossEntropyLoss.html).
